tests/scryer/issues.rs

```diff
@@ -25,3 +25,10 @@ fn issue2588_load_html() {
 fn call_qualification() {
     load_module_test("tests-pl/issue2361-call-qualified.pl", "");
 }
+
+#[serial]
+#[test]
+#[cfg_attr(miri, ignore = "it takes too long to run")]
+fn issue2756_load_context_unreachable() {
+    load_module_test("tests-pl/issue2756-load-context.pl", "");
+}
```

Comment:
which issue is this? it seems 2756 is this PR?

Reply:
Yes, it's this PR ;)

I guess I shouldn't call it `issue`